### ISSUE TITLE
build-ca fail, rc2 binarys

### ISSUE THREAD

**Issue by addohm** [OpenVPN/easy-rsa#125]
I just downloaded 3.0.1 but had to use the binarys from rc2.  I don't know if there is a difference.  Either way, build-ca is failing.  This is admin command prompt window.

```
EasyRSA Shell
# ./easyrsa build-ca
./easyrsa[1215]: mktemp: not found
./easyrsa[1215]: mktemp: not found
Generating a 2048 bit RSA private key
.........+++
...............+++
writing new private key to ''
Invalid argument
2540:error:02001000:system library:fopen:system library:bss_file.c:402:fopen('',
'wb')
2540:error:20074002:BIO routines:FILE_CTRL:system lib:bss_file.c:404:

Easy-RSA error:

Failed to build the CA

EasyRSA Shell
```

**Comment by TinCanTech** [comment]
Confirmed.

`mktemp` is **not recognised** by Windows EasyRSA-3.0.1 `./easyrsa` 
and throws the error above.

`mktemp` is **not present** in Windows EasyRSA3.0.0.rc2 `./easyrsa` 
and so does **not** throw any error.

**Comment by TinCanTech** [comment]
This is now resolved by v3.0.3
